sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -1366,33 +1169,6 @@ object GpuCast {
     }
   }
 
-  def fixDecimalBounds(input: ColumnView,
```

Comment:
Why did you move these functions? It makes it much harder to review the code when things move around.